Containers/nextcloud/config/s3.config.php

```diff
@@ -16,12 +16,12 @@
         'port' => getenv('OBJECTSTORE_S3_PORT') ?: '',
         'storageClass' => getenv('OBJECTSTORE_S3_STORAGE_CLASS') ?: '',
         'objectPrefix' => getenv("OBJECTSTORE_S3_OBJECT_PREFIX") ? getenv("OBJECTSTORE_S3_OBJECT_PREFIX") : "urn:oid:",
-        'autocreate' => (strtolower($autocreate) === 'false' || $autocreate == false) ? false : true,
-        'use_ssl' => (strtolower($use_ssl) === 'false' || $use_ssl == false) ? false : true,
+        'autocreate' => (strtolower($autocreate) === 'false' || $autocreate === false) ? false : true,
+        'use_ssl' => (strtolower($use_ssl) === 'false' || $use_ssl === false) ? false : true,

```

Comment:
```suggestion
        'autocreate' => strtolower($autocreate) !== 'false',
        'use_ssl' => strtolower($use_ssl) !== 'false',
```

This condition was very hard to read and interpret.

My suggestion reflects the default value (true) but admins can set it to `"false"` to disable the corresponding config.

Reply:
Oops didn't see this PR before I did this one: #6903